index.php

```diff
@@ -1,5 +1,10 @@
 <?php
 
+
+$language = "zh_CN";
+putenv("LANG=".$language);
+setlocale(LC_ALL, $language);
```

Comment:
I am not able to get the result even changing the setlocale to utf8, which I am really have no idea why. Do you have a sample site that you have done successfully? Also if you don't mind can you push the change to a different branch so I can look at it.